core/task/class.go

```diff
@@ -41,6 +42,7 @@ type TaskClass info
 //   role info.
 type info struct {
 	Name        string                  `yaml:"name"`
```

Comment:
Is the Name field still used for other things besides unmarshaling?

Reply:
Nope, moved the unmarshalling directly to taskIdentifier.